internal/schema-registry/command_dek_subject_list.go

```diff
@@ -0,0 +1,61 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newDekSubjectListCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
```

Comment:
Should we also have `describe` command if we have a `list` command?

Reply:
a dek subject is just a normal schema subject. you can run `confluent sr subject describe` then